### PR TITLE
build(CodeCov): Enable CodeCov.

### PR DESCRIPTION
**Pull request by arriolac**
Enabling [CodeCov](https://codecov.io) on Travis so that code coverage reports can be generated and evaluated on a per-PR basis.

### CODE REVIEW

**Comment by barbeau** [comment]
One of the Travis builds timed out, so I just restarted it (the rest passed).

File: .travis.yml [hunk 1]
```diff
@@ -96,7 +96,7 @@ install:
   - source ~/.install-jdk-travis.sh
 
 script:
-  - if [ "$API" != "NONE" ]; then ./gradlew test check connectedCheck -x lint --stacktrace; fi
+  - if [ "$API" != "NONE" ]; then ./gradlew test check jacocoTestReport connectedCheck -x lint --stacktrace; fi
```
Comment (barbeau) [inline .travis.yml]: `test` executes the JUnit tests we have in the `library/src/test`, and `connectedCheck` executes the instrumented tests we have in `library/src/androidTest`. Do you know if `jacocoTestReport` covers the output of both of these?

Comment (arriolac) [reply .travis.yml]: `jacocoTestReport` will only evaluate the existence of a test (unit or instrumented test) for a particular set of classes (i.e. classes within the `:library` module). The actual result of the unit and instrumented tests are currently handled by Travis.

Example CodeCov result:
image

**Comment by arriolac** [comment]
I misread the CodeCov report and thought that reports weren't being uploaded which is true for `master` since this change hasn't landed on `master` yet 😅 That said, no further changes are needed. cc @barbeau

**Pull request merged**